third_party/llvm/llvm.BUILD

```diff
@@ -128,6 +129,125 @@ cmake_vars = {
     "RETSIGTYPE": "void",
 }
 
+# Polly config vars.
+polly_cmake_vars = {
+    "CUDALIB_FOUND": 0,
+    "GPU_CODEGEN": 0,
+}
+
+isl_cmake_vars = {
+    # Headers
+    "HAVE_DIRENT_H": 1,
+    "HAVE_DLFCN_H": 1,
+    "HAVE_ERRNO_H": 1,
+    "HAVE_EXECINFO_H": 1,
+    "HAVE_FCNTL_H": 1,
+    "HAVE_INTTYPES_H": 1,
+    "HAVE_PTHREAD_H": 1,
+    "HAVE_SIGNAL_H": 1,
+    "HAVE_STDINT_H": 1,
+    "HAVE_SYS_IOCTL_H": 1,
+    "HAVE_SYS_MMAN_H": 1,
+    "HAVE_SYS_PARAM_H": 1,
+    "HAVE_SYS_RESOURCE_H": 1,
+    "HAVE_SYS_STAT_H": 1,
+    "HAVE_SYS_TIME_H": 1,
+    "HAVE_SYS_TYPES_H": 1,
+    "HAVE_TERMIOS_H": 1,
+    "HAVE_UNISTD_H": 1,
+    "HAVE_ZLIB_H": 1,
+
+    # Features
```

Comment:
How did you generate these?  If there are subsets (like `# Features`) that have to be common with `cmake_vars` by construction, we should common them up.

Reply:
Copied the LLVM features from `cmake_vars` but added isl-specific features to the second one.